Build.cs

```diff
@@ -206,6 +206,16 @@ private void WriteEnemies()
                 SeedRNG();
                 ROMFuncs.ShuffleEnemies(RNG);
             }
+        }
+
+        /// <summary>
+        /// Update the gossip stone actor to not check mask of truth
+        /// </summary>
+        private void WriteFreeHints()

```

Comment:
Overall I do like this code better than apply a hack file that does who-knows-what. hack file contents would be `00 E0 AB 88 01 00 FF`